.travis.yml

```diff
@@ -2,10 +2,8 @@ language: node_js
 
 # https://github.com/nodejs/LTS
 node_js:
-  - "8"
```

Comment:
`8` is still in `Active LTS` and we should definitely test it + we need to test `10`

Reply:
v10 is tested as it is the current stable release used via `'node'`.

Reply:
I prefer to use explicit version, especially for consistency with `graphql-js` and other `graphql/*` repos.